.github/workflows/pypi-publish.yml

```diff
@@ -132,6 +137,7 @@ jobs:
           dist/*.tar.gz
 
     - name: Publish to Test PyPI
+      if: ${{ !inputs.skip_test_pypi }}
```

Comment:
@kesara you might need to use `fromJSON()` here since most of the internal vars in GitHub are implicitly turned into strings, which may yield surprising results when you expect booleans.

Here's my typical approach to solving this: https://github.com/ansible/awx-plugins/blob/cfcafa5/.github/workflows/ci-cd.yml#L787-L795. But some people just do `!= 'true'`.